grpc/sa-wrappers.go

```diff
@@ -1116,11 +1128,24 @@ func (sas StorageAuthorityServerWrapper) GetOrderForNames(
 func (sas StorageAuthorityServerWrapper) GetOrderAuthorizations(
 	ctx context.Context,
 	request *sapb.GetOrderAuthorizationsRequest) (*sapb.Authorizations, error) {
+
+	// Call the wrapper for the renamed version of this RPC
+	return sas.GetValidOrderAuthorizations(
+		ctx,
+		&sapb.GetValidOrderAuthorizationsRequest{
+			Id:     request.Id,
+			AcctID: request.AcctID,
+		})
+}
+
+func (sas StorageAuthorityServerWrapper) GetValidOrderAuthorizations(
+	ctx context.Context,
+	request *sapb.GetValidOrderAuthorizationsRequest) (*sapb.Authorizations, error) {
 	if request == nil || request.Id == nil {
```

Comment:
This seems like it should also be checking that `request.AcctID != nil`, perhaps this was left out when originally introduced?

Reply:
> perhaps this was left out when originally introduced?

Seems like it. Fixed in 23a04ea